apps/src/templates/sectionProgress/standards/StandardsReport.jsx

```diff
@@ -0,0 +1,206 @@
+    );
+    printWindow.document.write('<body onafterprint="self.close()">');
+    printWindow.document.write(printArea);
+    printWindow.document.write('</body></html>');
+    printWindow.document.close();
+  };
+
+  render() {
+    const {scriptFriendlyName} = this.props;
+    const linkToOverview = this.getLinkToOverview();
+    return (
+      <div>
+        <Button
+          onClick={this.printReport}
+          color={Button.ButtonColor.orange}
+          text={i18n.printReport()}
+          size={'narrow'}
+          style={styles.button}
+        />
+        <div id="printArea" style={styles.printView}>
+          <StandardsReportHeader
+            sectionName={this.props.sectionName}
+            teacherName={this.props.teacherName}
+          />
+          <div style={styles.reportContent}>
+            <h2 style={styles.headerColor}>{i18n.currentCourse()}</h2>
+            <StandardsReportCurrentCourseInfo
+              section={this.props.section}
+              scriptFriendlyName={this.props.scriptFriendlyName}
+              scriptData={this.props.scriptData}
+              scriptDescription={this.props.scriptDescription}
+              numStudentsInSection={this.props.numStudentsInSection}
+              numLessonsCompleted={this.props.numLessonsCompleted}
+              numLessonsInUnit={this.props.numLessonsInUnit}
+            />
+            {this.props.teacherComment && (
+              <div>
+                <h2 style={styles.headerColor}>{i18n.teacherComments()}</h2>
+                <p>{this.props.teacherComment}</p>
+              </div>
+            )}
+            <h2 style={styles.headerColor}>{i18n.CSTAStandardsPracticed()}</h2>
+            <StandardsProgressTable style={styles.table} />
+            <StandardsLegendForPrint />
+            <h2 style={styles.headerColor}>{i18n.standardsHowToForPrint()}</h2>
+            <SafeMarkdown
+              openExternalLinksInNewTab={true}
+              markdown={i18n.standardsHowToDetailsForPrint({
+                courseName: scriptFriendlyName,
+                courseLink: linkToOverview,
+                cstaLink: 'https://www.csteachers.org/page/standards'
+              })}
+            />
+            <h2 style={styles.headerColor}>{i18n.standardsGetInvolved()}</h2>
+            <SafeMarkdown
+              markdown={i18n.standardsGetInvolvedDetailsForPrint({
+                adminLink: pegasus('/administrator'),
+                parentLink: pegasus('/help'),
+                teacherLink: '/courses'
+              })}
+            />
+          </div>
+          <div style={styles.footer}>
+            <div style={styles.mission}>
+              <SafeMarkdown markdown={i18n.missionStatement()} />
+            </div>
+          </div>
+        </div>
+        <Button
+          onClick={this.printReport}
+          color={Button.ButtonColor.orange}
+          text={i18n.printReport()}
+          size={'narrow'}
+          style={styles.button}
+        />
```

Comment:
This button at the top and bottom of the page? Worth extracting if we're going to render twice?